Jenkinsfile

```diff
@@ -509,3 +544,36 @@ def archiveServerLogs(String mongoDbRealmContainerId, String commandServerContai
 def runCommand(String command){
   return sh(script: command, returnStdout: true).trim()
 }
+
+def build_jvm_linux() {
+    docker.build('jvm_linux', '-f packages/cinterop/src/jvmMain/linux/generic.Dockerfile .').inside {
+        sh """
+           cd packages/cinterop/src/jvmMain/linux/
+           rm -rf build-dir
+           mkdir build-dir
+           cd build-dir
+           cmake ..
+           make -j8
+        """
+
+        stash includes:'packages/cinterop/src/jvmMain/linux/build-dir/core/src/realm/object-store/c_api/librealm-ffi.so,packages/cinterop/src/jvmMain/linux/build-dir/librealmc.so', name: 'linux_so_files', allowEmpty: true
+    }
+}
+
+def build_jvm_windows() {
+  def cmakeOptions = [
+        CMAKE_GENERATOR_PLATFORM: 'x64',
+        CMAKE_BUILD_TYPE: 'Release',
+        REALM_ENABLE_SYNC: "ON",
```

Comment:
Maybe add a comment here and a reference to an issue about combing CMake options?